lib/jekyll-redirect-from/generator.rb

```diff
@@ -5,6 +5,9 @@ class Generator < Jekyll::Generator
     safe true
     attr_reader :site, :redirects
 
+    CONFIG_KEY = "redirect_from".freeze
+    JSON_KEY = "json".freeze
```

Comment:
Please remove these constants.
This plugin already requires at least Ruby 2.3 and has all string literals frozen by default.

Calling `"redirect_from"` even a 100 times is not going to impact performance as it'd return the same Ruby object each time.